paf-mvp-frontend/tests/test-cases/queue.test.ts

```diff
@@ -0,0 +1,88 @@
+import { TestWatcher } from 'jest';
+import { isExportDeclaration } from 'typescript';
+import { Command, CommandQueue, processCommands } from '../../src/utils/queue';
+
+class CommandMocker {
+  public called = false;
+
+  func(): Command {
+    return () => {
+      this.called = true;
+    };
+  }
+}
+
+describe('queue', () => {
+  let cmd1: CommandMocker;
+  let cmd2: CommandMocker;
+  let cmd3: CommandMocker;
+  const lib = {
+    queue: [] as CommandQueue,
+  };
+
+  beforeEach(() => {
+    cmd1 = new CommandMocker();
+    cmd2 = new CommandMocker();
+    cmd3 = new CommandMocker();
+  });
+
+  test('processCommands undefined do not crash', () => {
+    lib.queue = processCommands(undefined);
+  });
+
+  test('processCommands empty array do nothing', () => {
+    lib.queue = processCommands(lib.queue);
```

Comment:
while I understand the previous test only assesses that no exceptions are thrown, I think this one deserves some assertions.

Reply:
Well, what do you want to assert? 
The array is empty.

Reply:
Then I recommend:
```typescript
processCommands([]);
// Assert no exception thrown
```
(the affectation is useless, and explicit `[]` is more explicit than `lib.queue`

Reply:
For demonstration purposes, I prefer to keep it like that.